src/openms/source/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.cpp

```diff
@@ -140,6 +140,56 @@ namespace OpenMS
     background = avg_noise_level * nr_points;
   }
 
+  void MRMTransitionGroupPicker::calculateBgEstimationExact_(const MSChromatogram& chromatogram,
+      double best_left, double best_right, double peak_height, double & background, double & avg_noise_level)
+  {
+    // determine (in the chromatogram) the intensity at the left / right border
+    double intensity_left = 0.0;
+    double rt_apex = 0.0;
+    double intensity_right = 0.0;
+    for (MSChromatogram::const_iterator it = chromatogram.begin() + 1; it != chromatogram.end(); it++)
+    {
+      MSChromatogram::const_iterator it_prev = it;
+      it_prev--; //previous point
+
+      if (it->getMZ() >= best_left && it_prev->getMZ() <= best_left)
+      {
+        intensity_left = it_prev->getIntensity();
+      }
+      else if (it->getIntensity() >= peak_height && it_prev->getIntensity() <= peak_height)
+      {
+        rt_apex = (it->getMZ() + it_prev->getMZ())/2;
+      }
+      else if (it->getMZ() >= best_right && it_prev->getMZ() <= best_right)
+      {
+        intensity_right = it->getIntensity();
+      }
+    }
+
+    double intensity_max, intensity_min, rt_max, rt_min;
+    if (intensity_left >= intensity_right)
+    {
+      intensity_max = intensity_left;
+      rt_max = best_left;
+      intensity_min = intensity_right;
+      rt_min = best_right;
+    }
+    else 
+    {
+      intensity_max = intensity_right;
+      rt_max = best_right;
+      intensity_min = intensity_left;
+      rt_min = best_left;
```

Comment:
can you remove this variable? it breaks the build: 

/.../OpenMS/src/openms/source/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.cpp:169:50: error: variable ‘rt_min’ set but not used [-Werror=unused-but-set-variable]

     double intensity_max, intensity_min, rt_max, rt_min;
                                                  ^
cc1plus: all warnings being treated as errors
Linking CXX shared library ../../lib/libOpenMS.so